lib/index.js

```diff
@@ -211,19 +212,16 @@ function _createInvalidTitleRegex(legalTitleChars) {
 function _capitalizeTitle(result, siteInfo) {
     var nsCase = siteInfo.namespaces[result.namespace._id + ''].case;
     if (nsCase === 'first-letter') {
+        // TODO: Document where in core this is happening
```

Comment:
mb-strtoupper shouldn't be affected by locale
https://secure.php.net/manual/en/function.mb-strtoupper.php